edge/pkg/common/dbm/db.go

```diff
@@ -72,9 +73,20 @@ func InitDBManager() {
 	DBAccess.Using(dbName)
 }
 
+// CleanDBFile removes db file if it is neccessary
```

Comment:
nit: neccessary -> necessary 

This is causing CI failure.

Reply:
I will Thanks